### ISSUE TITLE
Unrecognized environment: ohos

### ISSUE THREAD

**Issue by Binlogo** [akesson/rustup-configurator#1]
When developing https://github.com/human-solutions/xcframework/pull/8, which is dependent on `rustup-configurator`, during build, it raised:
```
Error: building with xcframework

Caused by:
    0: checking rustup targets
    1: Rustup returned an invalid triple 'aarch64-unknown-linux-ohos' due to 'Unrecognized environment: ohos', please raise an issue at: https://github.com/akesson/rustup-target/issues
    2: Unrecognized environment: ohos
```

and

```
Error: building with xcframework

Caused by:
    0: checking rustup targets
    1: Rustup returned an invalid triple 'riscv32imafc-unknown-none-elf' due to 'Unrecognized architecture: riscv32imafc', please raise an issue at: https://github.com/akesson/rustup-target/issues
    2: Unrecognized architecture: riscv32imafc
```

**Comment by Binlogo** [comment]
Waiting for https://github.com/bytecodealliance/target-lexicon/pull/103 merge.
- [ ] Update `target-lexicon` dep

**Comment by akesson** [comment]
Thanks for the heads up. This should automatically go away once that PR has been merged and there's a new version of `target-lexicon`. But let's keep this issue open until we have confirmed it.

**Comment by Binlogo** [comment]
Resolved by #2
